tests/test_driver/firebase_storage/firebase_storage_e2e.dart

```diff
@@ -22,10 +21,7 @@ void setupTests() {
       );
       await FirebaseStorage.instance
           .useStorageEmulator(testEmulatorHost, testEmulatorPort);
-      await FirebaseAuth.instance.useAuthEmulator(testEmulatorHost, 9099);
-      if (FirebaseAuth.instance.currentUser == null) {
-        await FirebaseAuth.instance.signInAnonymously();
-      }
+
```

Comment:
How come this was removed?

Reply:
[this](https://invertase.slack.com/archives/GUS07F4HM/p1657203399704339). It works for web, macOS and iOS so not sure if it's needed anymore? can reinstate if it's still an issue for android.